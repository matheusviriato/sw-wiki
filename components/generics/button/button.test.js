import React from 'react';
import { render } from '@testing-library/react';
import Button from '.';

describe('Button tests', () => {
  describe('Render tests', () => {
    it('should match to snapshot', () => {
      const { baseElement } = render(<Button text="Some button" />);

      expect(baseElement).toMatchSnapshot();
    });
  });
});
