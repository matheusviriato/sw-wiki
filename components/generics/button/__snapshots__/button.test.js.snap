// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button tests Render tests should match to snapshot 1`] = `
<body>
  <div>
    <button
      class="css-1x9g7k4-btnBase-btnPrimary-Button"
    >
      Some button
    </button>
  </div>
</body>
`;
